temporal-sdk/src/main/java/io/temporal/internal/client/external/GenericWorkflowClientImpl.java

```diff
@@ -386,4 +386,15 @@ public GetWorkerTaskReachabilityResponse GetWorkerTaskReachability(
                 .getWorkerTaskReachability(req),
         grpcRetryerOptions);
   }
+
+  @Override
+  public ExecuteMultiOperationResponse executeMultiOperation(ExecuteMultiOperationRequest req) {
+    return grpcRetryer.retryWithResult(
+        () ->
+            service
+                .blockingStub()
+                .withOption(METRICS_TAGS_CALL_OPTIONS_KEY, metricsScope)
```

Comment:
Should we be including more metric tags here? `start` includes the workflow type and task queue, update includes the update name. It might make sense to include them here as well.

Reply:
Good idea! https://github.com/temporalio/sdk-java/pull/2199/commits/019690efdb6a005dbb5e89b4bd0bd22276e9ce4b